R/learner_mboost_surv_gamboost.R

```diff
@@ -17,7 +17,12 @@
 #'
 #' @export
 #' @template seealso_learner
-#' @template example
+#' @examples
+#' if (mlr3misc::require_namespaces(c("mlr3extralearners", "mboost", "pracma"), quietly = TRUE))  {
+#' # Define the Learner and set parameter values
```

Comment:
no parameter values are set.

Also, mboost and pracma are not required when I just want to construct the learner